iocore/cache/P_CacheInternal.h

```diff
@@ -49,9 +49,9 @@ struct EvacuationBlock;
 #endif
 
 #ifdef DEBUG
-#define DDebug(tag, fmt, ...) Debug(tag, fmt, ##__VA_ARGS__)
+#define DDbg(dbg_ctl, fmt, ...) Dbg(dbg_ctl, fmt, ##__VA_ARGS__)
```

Comment:
These seem pretty useful generally. Should we move the up next to Dbg?

Reply:
Perhaps, but these PRs are big enough without mission creep.